clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -9315,6 +9330,9 @@ ProcessDeclAttribute(Sema &S, Scope *scope, Decl *D, const ParsedAttr &AL,
   case ParsedAttr::AT_Error:
     handleErrorAttr(S, D, AL);
     break;
+  case ParsedAttr::AT_ExcludeFromExplicitInstantiation:
```

Comment:
What is the previous 'handle' function?  Typically with decl attributes we need have an entry on this switch.

Reply:
`exclude_from_explicit_instantiation` had no 'handle' function prior to this patch -- the only restriction was that it must be applied to the declaration of a variable, function, or class.

Reply:
It still needs to be 'added' to the AST though?  Where was that happening?

Reply:
In the default case of the switch statement (line 9142, `AL.getInfo().handleDeclAttribute(S, D, AL)`)